app/web/service/client.ts

```diff
@@ -93,10 +99,7 @@ const client = {
   threads: new ThreadsPromiseClient(URL, null, opts),
 };
 
-if (
-  process.env.NEXT_PUBLIC_COUCHERS_ENV !== "prod" &&
-  typeof window !== "undefined"
-) {
+if (!IS_PROD && typeof window !== "undefined") {
```

Comment:
I think you can just do window and not typeof window no? Because if it's not undefined itself then it definitely has a type.